deps/chakrashim/src/v8objecttemplate.cc

```diff
@@ -748,6 +748,20 @@ Local<Object> ObjectTemplate::NewInstance(Handle<Object> prototype) {
     }
   }
 
+  if (!objectTemplateData->className.IsEmpty()) {
+    jsrt::IsolateShim* iso = jsrt::IsolateShim::GetCurrent();
+    if (jsrt::DefineProperty(newInstanceRef,
+			     iso->GetToStringTagSymbolPropertyIdRef(),
+			     jsrt::PropertyDescriptorOptionValues::True, /* writable */
```

Comment:
Compatibility: if you want to do `process[Symbol.toStringTag] = "someOtherString";` you are allowed to. Similarly for any of the other classes that have `ClassName`s.